doc/sphinxext/gen_rst.py

```diff
@@ -8,19 +8,18 @@
 Files that generate images should start with 'plot'
```

Comment:
Very strange, thanks for catching.  In fact none of the changes were present, I wonder if I pulled the commit when rebasing.  Did the same copy from master again!